travis/licenses_golden/licenses_third_party

```diff
@@ -23889,6 +23889,79 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ====================================================================================================
 
 ====================================================================================================
+LIBRARY: pkg
+ORIGIN: ../../../dart/third_party/observatory_pub_packages/packages/async/LICENSE
+TYPE: LicenseType.bsd
+FILE: ../../../third_party/pkg/when/.travis.yml
+FILE: ../../../third_party/pkg/when/example/foo.json
+FILE: ../../../third_party/pkg/when/example/read_json_file.dart
```

Comment:
None of these files end up in any binaries. Please update the license script to skip past `.travis.yml` files and anything in `../../../third_party/pkg/when/example`.